charts/ingress-nginx/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: ingress-nginx
 # When the version is modified, make sure the artifacthub.io/changes list is updated
 # Also update CHANGELOG.md
-version: 4.3.0
-appVersion: 1.4.0
+version: 4.4.0
+appVersion: 1.5.0
```

Comment:
No need to bump `appVersion` here? That's only if we released Ingress NGINX `v1.5.0`.

Reply:
Updated